samples/charts/financial-chart/performance/Pages/FinancialChartPerformance.razor

```diff
@@ -0,0 +1,47 @@
+﻿@page "/FinancialChartPerformance"
+@* this namespace is required for Infragistics controls *@
+@using Infragistics.Blazor.Controls
+@using Samples.Services
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+
+<div class="igContainer">
+    <div class="igComponent">
+
+        @if (DataSource != null)
+        {
+            <FinancialChart Width="100%"
+                Height="100%"
+                ChartType=FinancialChartType.Line
+                Thickness=2
+                XAxisMode="FinancialChartXAxisMode.Ordinal"
+                YAxisMode="FinancialChartYAxisMode.Numeric"
```

Comment:
you probably do not need to use 
- "FinancialChartXAxisMode."
- "FinancialChartYAxisMode."